src/tracking-grievance-api.js

```diff
@@ -0,0 +1,26 @@
+import dotenv from 'dotenv';
+import express from 'express';
+import cors from 'cors';
+import { connect } from './configs/db.config.js'
+import { companyRouter } from './routes/company.route.js'
+
+dotenv.config();
+
+const app = express();
+
+app.use(cors());
+
+app.use(express.json());
+
+app.use(express.urlencoded({extended: true}));
+
+app.use("/company", companyRouter);
+
```

Comment:
```
const startTime = new Date();

app.get("/", async(req,res)=>{
     res.send(`The server is running since ${startTime`)
 });
```

This is so because - what you did will update the running time every time the route is hit, otherwise outside the route will just keep one variable initialisation when the app was started.